src/test/java/org/mapstruct/intellij/bugs/_148/AutoInheritConfigurationTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at https://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.intellij.bugs._148;
```

Comment:
Is this test for #148 or for #151?

In any case I think that this should be added in the `org.mapstruct.intellij.inspection` test package. You can add a new folder in the `testData/inspection` for the auto config if you want.

Reply:
Sounds good, will do! I created that package and test even before creating an issue in the github repo, so 148 was a placeholder at that point.